src/lib/flow/include/sol-flow-packet.h

```diff
@@ -135,6 +136,11 @@ struct sol_flow_packet *sol_flow_packet_new_direction_vector_components(double x
 int sol_flow_packet_get_direction_vector(const struct sol_flow_packet *packet, struct sol_direction_vector *direction_vector);
 int sol_flow_packet_get_direction_vector_components(const struct sol_flow_packet *packet, double *x, double *y, double *z);
 
+struct sol_flow_packet *sol_flow_packet_new_location(const struct sol_location *location);
+struct sol_flow_packet *sol_flow_packet_new_location_latlon(double lat, double lon);
```

Comment:
I'll do this but on another series. For now, I'll just make the location packet pass a `const struct sol_location`.